content/22.vaccines-trad.md

```diff
@@ -138,8 +141,10 @@ Even for a small number of hamsters that developed breakthrough infections after
 However, much higher rates of breakthrough infection and higher viral loads were observed when the hamsters were exposed to the Beta variant [@doi:10.1101/2021.11.12.468374].
 Reduced seroconversion and neutralizing antibody (nAb) titers were also observed against the Beta and Gamma variants [@doi:10.1101/2021.11.12.468374].
 As a result, a modified version of YF-S0, called YF-S0\*, was developed to include a modified spike protein intended to increase immunogenicity by including the full spectrum of amino acids found in the Gamma VOC as well as stabilizing the S protein's conformation [@doi:10.1101/2021.11.12.468374].
-No breakthrough infections were observed following vaccination with YF-S0 and exposure to the index strain and the Alpha, Beta, Gamma, and Delta variants [@doi:10.1101/2021.11.12.468374].
-YF-S0\* also reduced viral shedding for several VOC relative to YF-S0 [@doi:10.1101/2021.11.12.468374], and it was also observed to produce significantly more nAbs against the Omicron variant [@doi:10.1101/2021.11.12.468374].
+The updated vaccine was again tested in Syrian golden hamsters.
+No breakthrough infections were observed following vaccination with YF-S0\* and exposure to the index strain and the Alpha, Beta, Gamma, and Delta variants [@doi:10.1101/2021.11.12.468374].
+YF-S0\* also reduced the infectious viral load in the lungs of several VOCs (Alpha, Beta, Gamma, and Delta) relative to a sham comparison [@doi:10.1101/2021.11.12.468374], and the likelihood of the Delta variant spreading to unvaccinated co-housed hamsters was significantly reduced by YF-S0/* [@doi:10.1101/2021.11.12.468374].
```

Comment:
Does the direction of the / matter in YF-SO?

Reply:
It does-- good catch! It's called an escape character and it's used to make sure characters that usually have a special meaning get rendered literally as text (so we just want it to say S0\*, not to interpret \* as what it would mean in a programming context)\\

```suggestion
YF-S0\* also reduced the infectious viral load in the lungs of several VOCs (Alpha, Beta, Gamma, and Delta) relative to a sham comparison [@doi:10.1101/2021.11.12.468374], and the likelihood of the Delta variant spreading to unvaccinated co-housed hamsters was significantly reduced by YF-S0\* [@doi:10.1101/2021.11.12.468374].
```